src/NoFrixion.MoneyMoov/Models/Merchant/MerchantPisBankSetting.cs

```diff
@@ -0,0 +1,65 @@
+﻿// -----------------------------------------------------------------------------
+//  Filename: MerchantPisBankSetting.cs
+// 
+//  Description: Model representing an individual PIS bank setting.
+// 
+//  Author(s):
+//  Axel Granillo (axel@nofrixion.com)
+// 
+//  History:
+//  08 03 2023  Axel Granillo   Created, Remote, Mexico City, Mexico.
+// 
+//  License:
+//  Proprietary NoFrixion.
+// -----------------------------------------------------------------------------
+
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NoFrixion.MoneyMoov.Models;
+
+/// <summary>
+/// Represents an individual PIS bank setting.
+/// </summary>
+public class MerchantPisBankSetting
+{
```

Comment:
Maybe MerchantPayByBankSetting? While PIS makes sense to us it won't be immediately obvious to API users.